src/App.js

```diff
@@ -1,11 +1,124 @@
 import React, { Component } from 'react';
-import './App.css';
+import axios from 'axios';
+import MovieList from './MovieList';
+import SearchBar from './SearchBar';
 
 class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      movies: [],
+      savedMovies: [],
+      searchText: ''
+    };
+  }
+  componentDidMount() {
+    this.getPopularMovies();
+    this.getSavedMovies();
+  }
+  getSavedMovies() {
+    axios.get('http://localhost:4000/movies')
+      .then(resp => {
+        this.setState({
+          savedMovies: resp.data
+        });
+      });
+  }
+  getPopularMovies() {
+    const savedMoviesIdArr = this.state.savedMovies.map(savedMovie => savedMovie.id);
+    axios.get('http://api.themoviedb.org/3/movie/popular?api_key=2dba200e2682e0f8903ed87b9c9e02d1')
+      .then(resp => {
+        // Only return popular movies whos IDs are NOT found in my savedMovies
+        this.setState({
+          movies: resp.data.results.filter(resultsMovie =>
+            !savedMoviesIdArr[savedMoviesIdArr.indexOf(resultsMovie.id)])
```

Comment:
You could probably clean this up a bit,

`savedMoviesIdArr.indexOf(resultsMovie.id) === -1`

This indicates that the id is not in the array.